tasks/sep-dev-0/006-holocene-sys-cfg/README.md

```diff
@@ -0,0 +1,32 @@
+# Holocene Hardfork - Contract Upgrades
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Upgrades the `SystemConfig` contract for the Holocene hardfork.
+
+Contracts upgraded are included within the
+[op-contracts/v1.8.0-rc.1](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.1) release.
+
+## Pre-deployments
+
+- `SystemConfig` - `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
+
```

Comment:
What's the source of truth for this value, i.e. how can I verify this is correct?

Reply:
It was deployed prior to the upgrade. We don't update the addresses in `superchain-registry` until we've done the upgrade.

You can check the [code page](https://sepolia.etherscan.io/address/0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD#code) on etherscan to see the contract.